src/Components/Blazor/WebAssembly.Authentication/src/AuthenticationManager.cs

```diff
@@ -0,0 +1,446 @@
+
+            return JS.InvokeVoidAsync("location.replace", registerUrl);
+        }
+
+        private ValueTask RedirectToProfile() => JS.InvokeVoidAsync("location.replace", Navigation.ToAbsoluteUri(ApplicationPaths.RemoteProfilePath).PathAndQuery);
+
+        private string GetErrorMessage() => GetParameter("message");
+
+        private static void DefaultLoginFragment(RenderTreeBuilder builder)
+        {
+            builder.OpenElement(0, "p");
+            builder.AddContent(1, "Checking login state...");
+            builder.CloseElement();
+        }
+
+        private static void RegisterNotSupportedFragment(RenderTreeBuilder builder)
+        {
+            builder.OpenElement(0, "p");
+            builder.AddContent(1, "Registration is not supported.");
+            builder.CloseElement();
+        }
+
+        private static void ProfileNotSupportedFragment(RenderTreeBuilder builder)
+        {
+            builder.OpenElement(0, "p");
+            builder.AddContent(1, "Editing the profile is not supported.");
+            builder.CloseElement();
+        }
+
+        private static void DefaultLoginCallbackFragment(RenderTreeBuilder builder)
+        {
+            builder.OpenElement(0, "p");
+            builder.AddContent(1, "Completing login...");
+            builder.CloseElement();
+        }
+
+        private static RenderFragment DefaultLoginFailedFragment(string message)
+        {
+            return builder =>
+            {
+                builder.OpenElement(0, "p");
+                builder.AddContent(1, "There was an error trying to log you in: '");
+                builder.AddContent(2, message);
+                builder.AddContent(3, "'");
+                builder.CloseElement();
+            };
+        }
+
+        private static void DefaultLogoutFragment(RenderTreeBuilder builder)
+        {
+            builder.OpenElement(0, "p");
+            builder.AddContent(1, "Processing logout...");
+            builder.CloseElement();
+        }
+
+        private static void DefaultLogoutCallbackFragment(RenderTreeBuilder builder)
+        {
+            builder.OpenElement(0, "p");
+            builder.AddContent(1, "Processing logout callback...");
+            builder.CloseElement();
+        }
+
+        private static RenderFragment DefaultLogoutFailedFragment(string message)
+        {
+            return builder =>
+            {
+                builder.OpenElement(0, "p");
+                builder.AddContent(1, "There was an error trying to log you out: '");
+                builder.AddContent(2, message);
+                builder.AddContent(3, "'");
+                builder.CloseElement();
+            };
+        }
+
+        private static void DefaultLoggedOutFragment(RenderTreeBuilder builder)
+        {
+            builder.OpenElement(0, "p");
+            builder.AddContent(1, "You successfully logged out!");
```

Comment:
```suggestion
            builder.AddContent(1, "You are logged out.");
```

Otherwise it sounds a bit more chirpy than I'd expect... as if logging out is a real accomplishment :)